meetings/minutes/weekly-2021-01-01.md

```diff
@@ -0,0 +1,13 @@
+# Weekly report
+
+**2021-01-01**
+
+## Test & Bugfix
+
+* store: disable async commit and 1pc in local txn: https://github.com/pingcap/tidb/pull/21995
+* (internal to PingCAP) v4.0.10 transaction test checklist: https://docs.google.com/document/d/1lzTRe7XN5yPSTt5wAMSt59MPgccMdlEOp9Wo3bbkVRA/ 
```

Comment:
can we make it public? Otherwise, I'd like to remove this line.

Reply:
I'm afraid not. Let me remove it.